packages/lesswrong/components/sunshineDashboard/ModerationDashboard.tsx

```diff
@@ -90,18 +89,7 @@ const styles = (theme: ThemeType): JssStyles => ({
   },
 });
 
-const reduceCommentModeratorActions = (commentModeratorActions: CommentModeratorActionDisplay[]): CommentWithModeratorActions[] => {
-  const allComments = commentModeratorActions.map(action => action.comment);
-  const uniqueComments = uniqBy(allComments, comment => comment._id);
-  const commentsWithActions = uniqueComments.map(comment => {
-    const actionsWithoutComment = commentModeratorActions.filter(result => result.comment._id === comment._id).map(({ comment, ...remainingAction }) => remainingAction);
-    return { comment, actions: actionsWithoutComment };
-  });
-
-  return commentsWithActions;
-};
-
-const tabs = new TupleSet(['sunshineNewUsers', 'allUsers', 'moderatedComments'] as const);
+const tabs = new TupleSet(['sunshineNewUsers', 'allUsers', 'recentlyActive', 'moderatedComments'] as const);
```

Comment:
Are we removing the `moderatedComments` tab?

Reply:
Yes.